e2e/atlas/search_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build e2e atlas,search
+
+package atlas_test
+
+import (
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/mongodb/mongocli/e2e"
+)
+
+func TestSearch(t *testing.T) {
+	const clustersEntity = "clusters"
+	const searchEntity = "search"
+
+	clusterName, err := deployCluster()
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	defer func() {
+		if e := deleteCluster(clusterName); e != nil {
+			t.Errorf("error deleting test cluster: %v", e)
+		}
+	}()
+
+	cliPath, err := e2e.Bin()
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	t.Run("list", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			atlasEntity,
+			clustersEntity,
+			searchEntity,
+			"list",
+			"--clusterName="+clusterName,
+			"--db=test",
+			"--collection=test")
+
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
```

Comment:
Can we do a check on the response here?

Reply:
it would be an empty array so I don't know how helpful that'll be

Reply:
ah ok